gslib/commands/iam.py

```diff
@@ -645,7 +645,7 @@ def _SetIam(self):
       policy_it = itertools.repeat(protojson.encode_message(policy))
       self.Apply(
           _SetIamWrapper,
-          itertools.izip(
+          zip(
```

Comment:
The args can now (probably) fit on the same line, right?

Reply:
Oh, yes! I compacted these down a little in latest commit.